internal/storage/credit_card_repository.go

```diff
@@ -19,14 +19,14 @@ func NewCreditCardRepository(db *gorm.DB) CreditCardRepository {
 
 // All ...
 func (p *CreditCardRepository) All() ([]model.CreditCard, error) {
-	creditCards := []model.CreditCard{}
+	var creditCards []model.CreditCard
 	err := p.DB.Find(&creditCards).Error
 	return creditCards, err
 }
 
 // FindAll ...
 func (p *CreditCardRepository) FindAll(argsStr map[string]string, argsInt map[string]int) ([]model.CreditCard, error) {
-	creditCards := []model.CreditCard{}
+	var creditCards []model.CreditCard
```

Comment:
Same issue.

Reply:
If the record is not exist on db, then response will be
```
{
    "code": 404,
    "status": "Error",
    "message": "record not found",
    "errors": null
}
```
is not ok ?